modules/networking/default.nix

```diff
@@ -117,6 +128,10 @@ in
       ''}
 
       ${setNetworkServices}
+
+      ${optionalString (cfg.wakeOnLan != null) ''
+        systemsetup -setWakeOnNetworkAccess '${onOff cfg.wakeOnLan}' &> /dev/null
```

Comment:
The `systemsetup` stdout and stderr are silenced:
* The changed value is echoed on stdout, e.g. `setwakeonnetworkaccess: On`.
* An error is reported on stderr, but the command succeeds as reflected by the corresponding get command and System Settings. Example error: `2023-11-18 15:47:54.640 systemsetup[4589:139580] ### Error:-99 File:/AppleInternal/Library/BuildRoots/11aa8fb2-5f4b-11ee-bc7f-926038f30c31/Library/Caches/com.apple.xbs/Sources/Admin/InternetServices.m Line:379`

The above observations apply to the new `power` and `power.sleep` options as well.

Reply:
I see this matches the work done here: https://github.com/LnL7/nix-darwin/pull/781